module/questions/helper.js

```diff
@@ -842,4 +842,113 @@ module.exports = class QuestionsHelper {
     }) 
   }
 
+
+  /**
+   * Create copy of questions.
+   * @method
+   * @name createCopyOfQuestions
+   * @param {Array} questionIds - Array of question Id's         
+   * @returns {Object}  old and new Mapped question ids .  
+  */
+
+  static createCopyOfQuestions(questionIds = []) {
```

Comment:
@deepakr123 - Please rename the function to **"duplicate"**

Reply:
resolved